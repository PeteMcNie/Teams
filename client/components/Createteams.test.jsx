import React from 'react'

import { renderWithRedux } from '../testing/utils'

import CreateTeams from './Createteams'

test('CreateTeams component matched snapshot', () => {
  const { container } = renderWithRedux(<CreateTeams />)
  expect(container).toMatchSnapshot()
})
