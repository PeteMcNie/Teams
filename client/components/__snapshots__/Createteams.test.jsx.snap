// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateTeams component matched snapshot 1`] = `
<div>
  <div>
    <p
      class="heading"
    >
      Pick team members
    </p>
    <div>
      <img
        class="loading"
        src="/images/loading.svg"
      />
    </div>
    <div
      class="link"
    >
      <a
        href="/addplayers"
      >
        Add New People
      </a>
    </div>
    <div>
      <div
        class="link"
      >
        <a
          class="home"
          href="/"
        >
          Home
        </a>
      </div>
    </div>
  </div>
</div>
`;
